application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -45,7 +46,11 @@ public void onMessageReceived(MessageReceivedEvent event) {
         }
 
         if (messageHasNoMediaAttached(message)) {
-            message.delete().flatMap(any -> dmUser(message)).queue();
+            message.delete().flatMap(any -> dmUser(message)).queue(any -> {
+            }, err -> {
+                pingUserAobutMedia(message)
+                    .queue(res -> res.delete().queueAfter(1, TimeUnit.MINUTES));
```

Comment:
what is `res`? bad name

Reply:
Shall I make it result or success ?

Reply:
none, id say `warningMessage`

Reply:
ah it stands for result. i thought its resource - see, thats why abbreviating is **bad**.
call it what it is, namely the `notificationMessage` or similar

Reply:
I update it with `notificationMessage `